### PR TITLE
[Docs] Add tag that stripe is outdated and add SCA note

### PR DESCRIPTION
**Pull request by Tomanhez**
| Q               | A
| --------------- | -----
| Branch?         | 1.4
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | no
| Deprecations?   | no
| License         | MIT

Screenshot 2019-10-14 at 16 02 31

### CODE REVIEW

File: docs/cookbook/payments/stripe.rst [hunk 1]
```diff
@@ -1,6 +1,20 @@
+.. warning::
+
+    On September 14, 2019 UE introduces effort `Strong Customer Authentication (SCA) <https://en.wikipedia.org/wiki/Strong_customer_authentication>`_ requirement.
```
Comment (CoderMaggie) [inline docs/cookbook/payments/stripe.rst]:
```suggestion
On September 14, 2019 the `Strong Customer Authentication (SCA) 
    <https://eur-lex.europa.eu/legal-content/EN/TXT/uri=uriserv:OJ.L_.2018.069.01.0023.01.ENG&toc=OJ:L:2018:069:TOC>`_ 
    requirement has been introduced.
```

File: docs/cookbook/payments/stripe.rst [hunk 2]
```diff
@@ -1,6 +1,20 @@
...
+    We can't offer you a good SCA solution at this time.
```
Comment (CoderMaggie) [inline docs/cookbook/payments/stripe.rst]:
```suggestion
    The implementation provided by Sylius core was not *SCA Ready* and has been deprecated.
    Please have a look at the [official documentation of Stripe regarding this topic](https://stripe.com/guides/strong-customer-authentication).
```

File: docs/cookbook/payments/stripe.rst [hunk 3]
```diff
@@ -1,6 +1,20 @@
...
+
 How to configure Stripe Credit Card payment?
 ============================================
 
+.. danger::
+
+   We're sorry but **this documentation section is outdated**. Please have that in mind when trying to use it.
+   You can help us making documentation up to date via Sylius Github. Thank you!
+
+..
```
Comment (CoderMaggie) [inline docs/cookbook/payments/stripe.rst]: unnecessary

File: docs/cookbook/payments/stripe.rst [hunk 4]
```diff
@@ -1,6 +1,21 @@
+.. warning::
+
+    On September 14, 2019 the `Strong Customer Authentication (SCA)
+    <https://eur-lex.europa.eu/legal-content/EN/TXT/uri=uriserv:OJ.L_.2018.069.01.0023.01.ENG&toc=OJ:L:2018:069:TOC>`_
+    requirement has been introduced.
+    The implementation provided by Sylius core was not *SCA Ready* and has been deprecated.
+    Please have a look at the [official documentation of Stripe regarding this topic](https://stripe.com/guides/strong-customer-authentication).
```
Comment (CoderMaggie) [inline docs/cookbook/payments/stripe.rst]:
```suggestion
    Please have a look at the (official documentation of Stripe regarding this topic)<https://stripe.com/guides/strong-customer-authentication>.
```

Didn't the formattion look suspicious to you? 🎉

File: docs/cookbook/payments/stripe.rst [hunk 5]
```diff
@@ -1,6 +1,21 @@
+.. warning::
+
+    On September 14, 2019 the `Strong Customer Authentication (SCA)
+    <https://eur-lex.europa.eu/legal-content/EN/TXT/uri=uriserv:OJ.L_.2018.069.01.0023.01.ENG&toc=OJ:L:2018:069:TOC>`_
+    requirement has been introduced.
+    The implementation provided by Sylius core was not *SCA Ready* and has been deprecated.
```
Comment (CoderMaggie) [inline docs/cookbook/payments/stripe.rst]:
```suggestion
    The implementation provided by Sylius Core was not *SCA Ready* and has been deprecated.
```

File: docs/cookbook/payments/stripe.rst (hunk 4 above)
Comment (Tomanhez) [reply docs/cookbook/payments/stripe.rst]: Sorry my mistake 🎃

**Comment by GSadee** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "sca-handling"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

**Comment by GSadee** [comment]
Thank you, Tomasz! :tada:

**Pull request merged**